src/util/EditionCardUtils.tsx

```diff
@@ -107,6 +107,30 @@ export default class EditionCardUtils {
     });
   }
 
+  static getAuthorsListByName(authors: string[]): JSX.Element[] {
```

Comment:
Is viaf still being used? If not, we should remove the above method.

Reply:
Viaf is still working.  If you click on an author who has a viaf, it does a search for "viaf" but displays the author name - you can see it when you click "Shakespeare" under https://digital-research-books-beta.nypl.org/work/023b92c2-c371-448c-8a10-4a9272ae0dee?featured=8576298

Reply:
you're probably going to want to add that to the edition detail pages as well - as you can seek if you click "Shakespeare" under https://sfr-bookfinder-front-end-git-sfr-914editionpageauthortitle-nypl.vercel.app/work/b9a174db-bf1f-4326-8270-8716117923e2?featured=6173984 it dies a slightly different search than if you click under the edition: https://sfr-bookfinder-front-end-git-sfr-914editionpageauthortitle-nypl.vercel.app/edition/6173984?featured=8861813